bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/handler/PulseaudioHandler.java

```diff
@@ -414,9 +423,12 @@ public String getHost() {
      * @return the port on which the pulseaudio server is listening for this sink
      * @throws InterruptedException when interrupted during the loading module wait
      */
-    public int getSimpleTcpPort() throws InterruptedException {
+    public int getSimpleTcpPort() throws IOException, InterruptedException {
```

Comment:
Please update the javadoc and check that the new exception is correctly handled by the caller.

Reply:
Yes the only call is from connectIfNeeded which already throws IOException